feat(cli/lint): add the `--rule` option (#2860)
biomejs/biome: CHANGELOG.md

### CLI

#### New features

> - Introduce a new valued oprion `--rule` for `biome lint` ([#58](https://github.com/biomejs/biome/issues/58)).
> 
>   This new option allows you to execute a single rule.
>   This is convenient to test a rule or apply the code fixes of a single rule.
> 
>   For example, you can execute the `style/useNamingConvention` rule on a set of files:
> 
>   ```shell
>   biome lint --rule=style/useNamingConvention index,js
>   ```
> 
>   And the napply its code fixes:
> 
>   ```shell
>   biome lint --rule=style/useNamingConvention --apply index,js
>   ```
> 
>   The option takes the rule configuration into account.
> 
>   The severity level of the executed rule is always set to its default,
>   i.e. `error` for a recommended rule or `warn` otherwise.
> 
>   The option is compatible with other options such as `--apply` and `--reporter`.
> 
>   Contributed by @Conaclos

Reviewer: ``````suggestion
- Add a new option `--rule` to the command `biome lint` ([#58](https://github.com/biomejs/biome/issues/58)).

  This new option allows you to execute a single rule. This option is convenient to test a rule or apply the code fixes of a single rule.

  For example, you can execute the `style/useNamingConvention` rule on a set of files:

  ```shell
  biome lint --rule=style/useNamingConvention src/index.js src/main.js
  ```

  If the rule has code action (autofix), you can use `--apply` to apply the fix:

  ```shell
  biome lint --rule=style/useNamingConvention --apply index.js src/main.js
  ```

  The option takes the rule configuration into account.

  The severity level of the executed rule is always set to its default,
  i.e. `error` for a recommended rule or `warn` otherwise.

  The option is compatible with other options such as `--apply`, `--apply-unsafe` and `--reporter`.

  Contributed by @Conaclos
``````

>  The option takes the rule configuration into account.

You might want to rephrase this. I can't suggest something because I don't know what it means, sorry about that.

### CLI

#### New features

- Add a new option `--rule` to the command `biome lint` ([#58](https://github.com/biomejs/biome/issues/58)).

  This new option allows you to execute a single rule. This option is convenient to test a rule or apply the code fixes of a single rule.

  For example, you can execute the `style/useNamingConvention` rule on a set of files:

  ```shell

>   biome lint --rule=style/useNamingConvention src/main.js

Reviewer: The reason why I suggested two files, is because the previous sentence says "set of files", hence the example shows a set of files.

I suggest reviewing the wording.
